### DISCUSSION TITLE
How to read cache-misses graph

### DISCUSSION THREAD

**Discussion by jhaprabhatt** [async-profiler/async-profiler#754] [Q&A] [answered]
Hi,
The flame graph looks similar to cpu or allocation graph. Any example of cache-misses and similar profiling ?

**Comment by krzysztofslusarski** [comment, marked as answer]
One is in my tutorial: https://krzysztofslusarski.github.io/2022/12/12/async-manual.html#perf-cache

**Reply by jhaprabhatt** [reply]
Thank you
